evap/evaluation/migrations/0065_questionnaire_type.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.0.2 on 2018-03-12 18:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0064_questionnaire_index_rename'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='questionnaire',
+            options={'ordering': ('type', 'order', 'name_de'), 'verbose_name': 'questionnaire', 'verbose_name_plural': 'questionnaires'},
+        ),
+        migrations.AddField(
+            model_name='questionnaire',
+            name='type',
+            field=models.IntegerField(choices=[(0, 'Top questionnaire'), (1, 'Contributor questionnaire'), (2, 'Bottom questionnaire')], default=0, verbose_name='type'),
+            preserve_default=False,
+        ),
+        migrations.RunSQL(
+            "UPDATE evaluation_questionnaire SET type = 1 WHERE is_for_contributors = TRUE;",
+            reverse_sql="UPDATE evaluation_questionnaire SET is_for_contributors = TRUE WHERE type = 1;"
```

Comment:
i know we talked about this before, but upon thinking about this again, there is no good reason to use plain sql here i think. apart from necessitating two methods, the ORM version should be similarly succinct.

Reply:
done